core/src/test/java/org/elasticsearch/search/simple/SimpleSearchIT.java

```diff
@@ -109,6 +108,11 @@ public void testSimpleIp() throws Exception {
         assertHitCount(search, 1L);
     }
 
+    @Override
+    protected int numberOfShards() {
```

Comment:
can you add a comment why this is only one shard? I wonder why we have to do one shard here on all tests?